app/views/metrics/index.html.erb

```diff
@@ -6,3 +6,61 @@
 <% content_for :javascripts do %>
   <%= javascript_include_tag "metrics" %>
 <% end %>
+
+<% @define_metrics = false %>
+
+<div class="ui pointing secondary tabular menu">
+  <a class="item active" data-tab="watchlist">Watchlist</a>
+  <a class="item" data-tab="risk metrics">Risk Metrics</a>
+</div>
+<div class="ui tab active" data-tab="watchlist">
+  <% if @define_metrics then %>
+    <div class="top-bar">
+      <div class="ui checkbox"><input type="checkbox" name="select_all"/></div>
+      <button class="ui submit small button"><i class="mail outline icon"></i>CONTACT</button>
+      <button class="ui submit small button"><i class="check circle icon"></i>RESOLVE</button>
+      <div class="last-updated">
+        <h5> Last Updated 10/20/2020 at 11:50pm <i class="repeat link icon"></i></h5>
+      </div>
+    </div>
+  <% end %>
+  <br>
+  <div class="ui grid">
+    <div class="three wide column">
+      <div class="ui vertical fluid tabular menu">
+        <a class="item active" data-tab="new_tab">New</a>
+        <a class="item" data-tab="contacted_tab">Contacted</a>
+        <a class="item" data-tab="resolved_tab">Resolved</a>
+        <a class="item" data-tab="archived_tab">Archived</a>
+      </div>
+    </div>
+    <div class="thirteen wide column">
+      <% if @define_metrics then %>
+        <div class="ui tab segments active" data-tab="new_tab">
+          <%= render partial: 'student_row', locals: {student_name: "Jane Doe", grace_days: 3, low_scores: 2} %>
```

Comment:
I think the student row looks great, it works like a component and is separate from the main watchlist, but I am concerned about using render partial in rendering the rows themselves. Because render partial is carried out on the server side, not the client side, as discussed previously in our hacking session.  This is an issue as it would mean that a refresh of the page is needed whenever an entry is refreshed / resolved / contacted

Reply:
would be good to collaborate with @ashleyzhang to resolve this issue

Reply:
This will be handled by @ashleyzhang  PR, this PR is for simplified frontend design.